.chloggen/session_change_event.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: 'session'
```

Comment:
I have no idea if this `component` is real or correct.

Reply:
I think @joaopgrassi's tooling infers components and generates them.

Could create a YAML in the model for this event with just the name?  Then you can have a (probably empty) snippet of the form:

```
<!-- semconv session.change_event -->
<!-- endsemconv -->
```

Reply:
Do we think that's worth doing now, or wait until the event tooling is baked?

Reply:
We already have a `session.yaml` in the registry` https://github.com/open-telemetry/semantic-conventions/blob/main/model/registry/session.yaml so this is correct already :)